Week3/MAKEME.md

```diff
@@ -41,6 +45,12 @@ In order to speed up development it's wise to know how to use a CSS framework. I
 
 ### 3. Code along
 
+If you need more practice with Git, please follow these links and follow the instructions.
```

Comment:
Perfect! Question: do you think tutorial #3 should still be concluded? (as you've mentioned you found it too advanced for students)

Reply:
I think we can keep it, because some (smart) students might want to play with Git. We can make it optional.